curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-042.md

```diff
@@ -0,0 +1,222 @@
+---
+id: 6145f8f8bcd4370f6509078e
+title: Part 42
+challengeType: 0
+dashedName: part-42
+---
+
+# --description--
+
+Within the `address` element, add the following:
+
+```html
+freeCodeCamp<br />
+San Fransisco<br />
+California<br />
+USA
+```
+
+You can visit, but you might not find anything...
+
+# --hints--
+
+You should add the above text including the `<br />` tags to the `address` element.
+
+```js
+assert.equal(document.querySelector('address')?.innerText, 'freeCodeCamp\nSan Fransisco\nCalifornia\nUSA');
```

Comment:
```suggestion
assert.equal(document.querySelector('address')?.innerText, 'freeCodeCamp\nSan Francisco\nCalifornia\nUSA');
```
Next parts are also affected

Reply:
This makes me sad, because I just recently found the `Califormia` typo: https://github.com/freeCodeCamp/freeCodeCamp/pull/43244/commits/22e45c3a8356051031c63ad89613ade99c80c9f8#diff-06d834488d682fdbf0efed92bd7152d572432d9df10fbb20212c2dcd26ef6392L168

Reply:
I'm quite okay with freeCodeCamp being based in San Fransisco, Califormia